locales/en-US/messages.properties

```diff
@@ -197,6 +197,8 @@ navAllowJSTitle=Toggle running JavaScript code in the preview
 navPreviewTitle=View the Preview
 navViewTutorialTitle=View the Project Tutorial
 navDOMElementInspectorIconTitle=Toggle Element Inspector
+navToggleAutoUpdateTitle=Toggle Preview Auto-refresh
+navToggleAutoUpdateLabel=Auto
```

Comment:
Let's move this specific label here: https://github.com/mozilla/thimble.mozilla.org/pull/1466/files#diff-aef074d5302a863440aa549e7a2fd981R107